Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Search.hs

```diff
@@ -326,3 +307,24 @@ mkQuoteInfo fromLoc toLoc startTime DQuoteSpecialZone.QuoteSpecialZone {..} = do
     { quoteId = id,
       ..
     }
+
+farePolicyHelper :: (HasField "maxAllowedTripDistance" a (Maybe Meters), HasField "minAllowedTripDistance" a (Maybe Meters)) => Meters -> [a] -> [a]
```

Comment:
I think this naming is not very clear; `filterFarePolicy` or `filterFarePolicyByDistance` would be better